src/components/element-editor/conditional-rendering/conditional-rendering.tsx

```diff
@@ -21,6 +23,7 @@ export const ConditionalRendering = () => {
     const element = focusedElement.element;
     const [childEntryValues, setChildEntryValues] =
         useState<IConditionalRendering[]>();
```

Comment:
default to array instead of setting empty array on line 52
```suggestion
        useState<IConditionalRendering[]>([]);
```